ietf-layer0-types.yang

```diff
@@ -2062,14 +2041,33 @@ module ietf-layer0-types {
       description
         "This parameter indicates the transmitter frequency fine
         tuning steps e.g 3.125GHz or 0.001GHz.";
-    } 
+    }
   } // grouping transmitter-tuning-range
 
   grouping common-all-modes {
     description
       "Common attributes used by all transceiver's modes.";
 
-    uses transmitter-tuning-range;
+    container transmitter-tuning-range {
+      config false;
+      description
+        "Transmitter tuning range (f_tx-min, f_tx-max)";
+      uses transmitter-tuning-range;
```

Comment:
I definitely prefer the container solution chosen now with tuning range parameters placed into a dedicated container instead of the flat solution present before. +1